packages/signaldb/src/SyncManager/index.ts

```diff
@@ -242,7 +236,7 @@ export default class SyncManager<
    * @param options.force If true, the sync process will be started even if there are no changes and onlyWithChanges is true.
    * @param options.onlyWithChanges If true, the sync process will only be started if there are changes.
    */
-  public async sync(name: string, options: { force?: boolean, onlyWithChanges?: boolean } = {}) {
+  private async sync(name: string, options: { force?: boolean, onlyWithChanges?: boolean } = {}) {
```

Comment:
I think it's better to leave the `sync` method as it is and just toggle the pause status of the collection with the `startSync`/`pauseSync` methods. Otherwise it isn't possible to trigger a manual sync while the remote change listener is paused.